src/System.Management.Automation/namespaces/RegistryWrapper.cs

```diff
@@ -17,29 +17,31 @@
 
 namespace Microsoft.PowerShell.Commands
 {
+
+#nullable enable
     internal interface IRegistryWrapper
     {
-        void SetValue(string name, object value);
+        void SetValue(string? name, object value);
 
-        void SetValue(string name, object value, RegistryValueKind valueKind);
+        void SetValue(string? name, object value, RegistryValueKind valueKind);
```

Comment:
What does it mean to set a null-named key in the registry?

Reply:
From docs: https://docs.microsoft.com/en-us/dotnet/api/microsoft.win32.registrykey.setvalue?view=net-5.0#Microsoft_Win32_RegistryKey_SetValue_System_String_System_Object_
> A registry key can have one value that is not associated with any name. When this unnamed value is displayed in the registry editor, the string "(Default)" appears instead of a name. To set this unnamed value, specify either null or the empty string ("") for name.